
<td align="center" class="tbcolor10" colspan="2" valign="top">
<table border="0" cellpadding="0" height="100%" width="100%">
<tbody><tr><td align="right" height="35px" style="font-size:9pt">
<a href="javascript:window.close()" title="关闭本窗口">关闭窗口</a>｜<a href="print.asp?id=482144" target="_blank" title="打印本页内容">打印张贴版</a>    

</td></tr>
<tr>
<td align="center" height="500" valign="top">
<table border="0" cellpadding="4" cellspacing="0" height="100%" style="border-collapse: collapse;word-break:break-all;" width="86%">
<tbody><tr>
<td align="center" height="80px"><span style="font-family:方正小标宋简体;font-size: 25px;color: #C70E5C">【学术讲座】DeepST: A novel graph self-supervised contrastive ...</span></td>
</tr>
<tr>
<td align="center" height="50px" style="font-size: 9pt"><font color="#808080">电子与信息工程学院　2022/10/25 14:57:00</font><br/> 
<font color="#F8F8F8" style="font-size:9pt">（钟沛基 2022280095）</font></td>
</tr>
<tr>
<td height="300" valign="top">
<p class="MsoNormal"><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin"><b>报告题目</b>：</span><span lang="EN-US">DeepST: A novel graph self-supervised contrastive learning method
for spatially informed clustering, integration, and deconvolution of spatial
transcriptomics<o:p></o:p></span></p><p class="MsoNormal"><span lang="EN-US"><span style="font-family: 宋体;"><b>报告嘉宾</b>：</span></span><span style="font-family: 宋体;">龙亚辉，新加坡科技研究局（</span><span lang="EN-US">A*STAR</span><span style="font-family: 宋体;">）博士</span></p><p class="MsoNormal"><span style="font-family: 宋体;"><b>邀请人</b>：欧阳乐 副教授</span></p>
<p class="MsoNormal"><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin"><b>报告时间</b>：</span><span lang="EN-US">2022</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">年</span><span lang="EN-US">10</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">月</span><span lang="EN-US">25</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">日（周二）下午</span><span lang="EN-US">3:00-4:00<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin"><b>报告地点</b>：</span><span lang="EN-US">ZOOM</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">，</span><span lang="EN-US">Meeting ID: 897 7383 3653<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin"><b>报告摘要</b>：</span><span lang="EN-US">Advances in spatial transcriptomics technologies has enabled gene
expression profiling of tissues while retaining the spatial context. To
effectively exploit the data, spatially informed analysis tools are required
for three key tasks, spatial clustering, multi-sample integration, and cell
type deconvolution. Here, we present DeepST, a novel graph self-supervised
contrastive learning method that incorporates spatial location information and
gene expression profiles to accomplish all three tasks in a streamlined process
and outperform existing methods in each task. DeepST combines graph neural
networks with self-supervised contrastive learning to learn informative and
discriminative spot representation by minimizing the embedding distance between
spatially adjacent spots and vice versa. With DeepST, we achieved 10% higher
clustering accuracy on multiple datasets than competing methods, and better
delineated the fine-grained structures in tissues such as the brain and embryo.
Moreover, DeepST is the only method that can jointly analyze multiple tissue
slices in both vertical and horizontal integration while correcting for batch
effects. Lastly, compared to other methods, DeepST</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;mso-ascii-theme-font:minor-latin;
mso-fareast-theme-font:minor-fareast;mso-hansi-font-family:Calibri;mso-hansi-theme-font:
minor-latin">’</span><span lang="EN-US">s cell type deconvolution showed higher
accuracy on simulated data and better captured spatial niches such as germinal
centers of the lymph node in experimentally acquired data. We further
demonstrated that DeepST can recover immune cell distribution in the different
regions of breast tumor tissue and reveal spatial niches with exhausted tumor
infiltrating T cells. Our results show that DeepST is widely applicable to a
broad range of tissue types and technology platforms. It offers a streamlined,
user friendly and computationally efficient tool for characterizing tissue
complexity and gaining biological insights into spatial organization and
cell-cell communication within tissues.<o:p></o:p></span></p>
<p class="MsoNormal"><span lang="EN-US"><o:p> </o:p></span></p>
<p class="MsoNormal"><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin"><b>报告人</b>：龙亚辉，新加坡科技研究局（</span><span lang="EN-US">A*STAR</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">）博士后。博士毕业于湖南大学信息科学与工程学院，师从骆嘉伟教授。主要研究领域为空间转录组学、深度学习、生物信息学等。</span><span lang="EN-US">2019</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">年至</span><span lang="EN-US">2021</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">年期间，曾赴新加坡</span><span lang="EN-US">Nanyang Technological University &amp; A*STAR</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;mso-ascii-theme-font:minor-latin;
mso-fareast-theme-font:minor-fareast;mso-hansi-font-family:Calibri;mso-hansi-theme-font:
minor-latin">访问学习。在博士期间，曾多次获得博士校长奖学金、国家奖学金、优秀博士研究生等荣誉。目前以第一作者在</span><span lang="EN-US">Bioinformatics, Briefings in Bioinformatics, IEEE Journal of
Biomedical and Health Informatics (JBHI), IEEE/ACM Transactions on
Computational Biology and Bioinformatics (TCBB), and European Conference on
Computational Biology (ECCB)</span><span style="font-family:宋体;mso-ascii-font-family:
Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">等</span><span lang="EN-US">SCI</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">期刊和国际会议发表论文</span><span lang="EN-US">10</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;
mso-ascii-theme-font:minor-latin;mso-fareast-theme-font:minor-fareast;
mso-hansi-font-family:Calibri;mso-hansi-theme-font:minor-latin">余篇。现担任</span><span lang="EN-US">Bioinformatics, Briefings in Bioinformatics, Neurocomputing</span><span style="font-family:宋体;mso-ascii-font-family:Calibri;mso-ascii-theme-font:minor-latin;
mso-fareast-theme-font:minor-fareast;mso-hansi-font-family:Calibri;mso-hansi-theme-font:
minor-latin">等多个国际学术期刊审稿人。</span><span lang="EN-US"><o:p></o:p></span></p><p class="MsoNormal" style="text-align: right;"><font face="宋体">电子与信息工程学院</font></p><p class="MsoNormal" style="text-align: right;"><font face="宋体">2022年10月25日</font></p></td>
</tr>
<tr><td height="0" style="font-size: 9pt"></td></tr>
<tr><td align="right" style="font-size: 9pt"> </td></tr>
</tbody></table>
</td>
</tr>
</tbody></table>
</td>
